package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@balancer/frontend-v2",
-  "version": "1.112.8",
+  "version": "1.112.9",
```

Comment:
Why is the version bumped?

Reply:
oh whoops I merged deploy into this branch instead of develop -_-

Reply:
Will fix